cni.go

```diff
@@ -140,6 +141,19 @@ func (c *libcni) Status() error {
 	if len(c.networks) < c.networkCount {
 		return ErrCNINotInitialized
 	}
+
+	for _, dir := range c.pluginDirs {
```

Comment:
nit.. per discussion offline we could/should do this check on load instead.. TODO:  consider if need to add a listener on the plugindirs to know when to reload as we do on the config path

Reply:
hesitation on doing to much here given the better design coming with the kni/cni daemons..  might want to ignore the TODO for now